README.md

```diff
@@ -170,12 +171,21 @@ Immediately triggers a new search for the given text
 
 - text `String` A word of sequence of words to search for
 
-#### `search.playResult(result)`
+**Returns:**
+- retVal `Promise` - A promise that will resolve with the results of the search you just performed.
+The promise will be rejected with a timeout error if the search takes too long.
+
+#### `search.playResult(searchText, result)`
 Immediately attempts to play the given result.  If we fail to play the given result
 an error will be thrown.
 
+- searchText `String` The search text that causes the search result to appear
```

Comment:
Hmmm, IMO it's "single behavior" is to play a search result and it will fail to do that if we are not on the correct search results page.

You could almost say it is a dependency of `playResult` that we are on the correct search page, why force the user to handle this when we can handle it internally?  This is part of the reason I added `searchText` to the `SearchResults` object.